kubernetes/deployment-production.tmpl

```diff
@@ -332,14 +348,26 @@ spec:
                   name: caesar-production-env-vars
                   key: SIDEKIQ_WEB_USERNAME
 ---
+apiVersion: autoscaling/v1
+ kind: HorizontalPodAutoscaler
+ metadata:
+   name: caesar-production-sidekiq
+ spec:
+   scaleTargetRef:
+     apiVersion: apps/v1
+     kind: Deployment
+     name: caesar-production-sidekiq
+   minReplicas: 2
+   maxReplicas: 8
```

Comment:
do we really want up to 8 sidekiq nodes - is that what the ASG was in AWS??

Reply:
Ah, no that was a copy/paste that I forgot to change.